docs/testrunner/testrunner_run-template.md

```diff
@@ -38,6 +38,7 @@ testrunner run-template [flags]
       --testruns-chart-path string         Path to the testruns chart.
       --timeout int                        Timout in seconds of the testrunner to wait for the complete testrun to finish. (default 3600)
       --tm-kubeconfig-path string          Path to the testmachinery cluster kubeconfig
+      --version-matrix                     Run the testrun with all available versions of the cloudprovider.
```

Comment:
No strong opinion, but maybe `--all-k8s-versions` or `--test-all-versions` would be better.

Reply:
changed to `all-k8s-version` for better understanding, but internally mapped to `makeVersionMatrix`.